docs/source/archive/gsoc/reports/2024/scancode_toolkit_swastkk.rst

```diff
@@ -0,0 +1,69 @@
+========================================================================
+Compute summary for all detected packages.
+========================================================================
+
+
+| **Organization:** `AboutCode <https://aboutcode.org>`_
+| **Project:** `Scancode Toolkit <https://github.com/aboutcode-org/scancode-toolkit>`_
+| **Mentee:** `Swastik Sharma (swastkk) <https://github.com/swastkk>`_
+| **Mentors:** Philippe Ombredanne, AyanSinhaMahapatra, AvishrantSh, Jonathan Yang, Jay Kumar
+
+Overview
+--------
+
+Previously, we computed the summary at the codebase level, which included elements like the
+`license_clarity_score`, `declared_holder`, `other_license_expressions`, and more.
+This project aims to improve scanning accuracy by computing summaries and license clarity scores for
+each package and its files, rather than for the entire scan. This involves enhancing package models
+and ensuring accurate attribute collection across all package ecosystems.
+
+Implementation
+--------------
+
+All the work I did is contained in `this single PR <https://github.com/aboutcode-org/scancode-toolkit/pull/3792>`_.
+I added a new command-line option called ``--package-summary`` that users can employ to obtain
+a package-level summary within a single codebase. The package level summary involves the
+``license_clarity_score`` calculation and population of package attributes like ``copyright``,
+``holder``, ``other_license_expression``, ``notice_text``. This option must be called
+with ``--classify`` option that helps ScanCode further classify scanned files/directories,
+to determine whether they fall in these categories `legal`, `readme`, `top-level`, `manifest`
+& ``--package`` or ``-p`` option detects various package manifests, lockfiles and
+package-like data and then assembles codebase level packages and dependencies from
+these package data detected at files. Also tags files if they are part of the packages.
+
+This change allows users to get the more refined summary for each individual package
+that is present in a codebase. Also this feature improves the package assembly for
+various package ecosystems like npm, python-whl, rust, rubygems etc.
+
+
+Finally, all these changes are tested through multiple unit tests validating both correct
+behavior and error handling as needed.
+
+Post GSoC
+---------
+
+I would like to merge this PR into Scancode Toolkit, hopefully allowing users to leverage
+this feature to expand their package/codebase scanning capabilities.
+
+Links
+-----
+
+`Project idea <https://github.com/aboutcode-org/aboutcode/wiki/GSOC-2024-Project-Ideas#compute-summary-for-all-detected-packages>`_
```

Comment:
This should be entries in an unordered list like the mentors one. https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks